microsoft-edge/webview2/concepts/windowed-vs-visual-hosting.md

```diff
@@ -1,436 +1,185 @@
-
-##### [WinRT/C#](#tab/winrtcsharp)
-
-* [CoreWebView2Controller.CoreWebView2 Property](/microsoft-edge/webview2/reference/win32/icorewebview2controller#get_corewebview2)
-* [CoreWebView2Controller.Close Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2controller#close)
-* [CoreWebView2Environment.CreateCoreWebView2ControllerAsync Method](/microsoft-edge/webview2/reference/winrt/microsoft_web_webview2_core/corewebview2environment#createcorewebview2controllerasync)
-
-##### [Win32/C++](#tab/win32cpp)
-
-* [ICoreWebView2Controller::CoreWebView2 property](/microsoft-edge/webview2/reference/win32/icorewebview2controller#get_corewebview2)<!--no put-->
-* [ICoreWebView2Controller::Close method](/microsoft-edge/webview2/reference/win32/icorewebview2controller#close)
-* [ICoreWebView2Environment::CreateCoreWebView2Controller method](/microsoft-edge/webview2/reference/win32/icorewebview2environment#createcorewebview2controller)
-
----
-
-
-<!-- ====================================================================== -->
-## Windowed hosting
-
-Windowed hosting can be described as an HWND that stores information.  You can have multiple HWNDs in your app that will each be used as a WebView component to access web content.  Some of the Output/Input commands are handled for you by the framework you choose; however, you will still need to handle window management.
+#### Advantages
 
-Benefits for Windowed hosting include:
+* Windowed hosting allows for a solution that quickly displays web content without having to implement functionality for inputs, outputs, and accessibility.
 
-* Zooming and rasterization scales (for example, menus, context menus, and so on) will also automatically scale to the app's parent `HWND`. It also handles how the WebView manages being focused and tabbing in/out of itself when it reaches the final element.
+* Owned and child windows (such as menus and context menus) automatically scale to match the app's parent `HWND` scaling.
 
-* The app handles keyboard accelerators and keyboard shortcuts when in the WebView. For example, **Ctrl+C** in a WebView will know that you are trying to copy content in the WebView, and not pressing **Ctrl** and **C** separately.
+* Windowed hosting handles how the WebView2 control manages being focused and tabbing in or out of itself when pressing **Tab** reaches the final element.
 
-* You don't have to manage the various composition-based rendering (for example, Inputs, Outputs, and Accessibility controls) if you don't want to.
+* The app handles keyboard accelerators and keyboard shortcuts when focus is on the WebView2. For example, pressing **Ctrl+C** in a WebView2 will be interpreted as trying to copy content in the WebView2, not interpreted as pressing **Ctrl** and **C** separately.<!-- todo: what is this item trying to convey?  handling keyboard accelerators is the same across all hosting modes, right? -->
```

Comment:
"Ctrl+C" - 
what is this list item trying to convey? 
handling keyboard accelerators is the same across all hosting modes, right?

Reply:
Yeah now that you mention it. This can probably be removed. I guess this may have been a benefit of WebView2 in general rather than a benefit of a particular hosting mode.

Reply:
removed list item